CMakeLists.txt

```diff
@@ -94,5 +94,5 @@ include(AzureVersion)
 add_subdirectory(sdk/core)
 add_subdirectory(sdk/identity)
 add_subdirectory(sdk/keyvault)
-add_subdirectory(sdk/storage)
+#add_subdirectory(sdk/storage)
```

Comment:
revert

Reply:
Need to disable until @Jinming-Hu fixes the breaking changes. That's how he has asked it multiple times :)

Reply:
I think that was OK early on in the development cycle, but let's avoid checking-in things like this to master where a large portion is not being built. I think the rename/move PRs should keep the entire repo building, and we shouldn't introduce artificial "who does the work" limitations.